pkg/helm/renderer.go

```diff
@@ -167,7 +179,19 @@ func (r *Renderer) renderWithHelm(componentName, raw, chartPath string) (map[str
 	options := chartutil.ReleaseOptions{
 		Name: componentName,
 	}
-	vals, err := chartutil.ToRenderValuesCaps(c, config, options, nil)
+
+	kubeVersion, err := r.k8sVersion()
+	if err != nil {
+		return nil, errors.Wrap(err, "setting Kubernetes version for Helm")
+	}
+
+	caps := &chartutil.Capabilities{
+		KubeVersion: &version.Info{
+			GitVersion: kubeVersion,
```

Comment:
Where does this ultimately get consumed? Are any of the other version-related fields in `version.Info` needed?

Reply:
It is consumed in the helm chart templates. None of the other fields are consumed within any charts in the stable repo.

Reply:
Here's an example: https://github.com/kubernetes/charts/blob/74a23f7efbb332ea873df89f09d28654c48f6615/stable/cockroachdb/templates/_helpers.tpl#L5